packages/rum/src/domain/record/serialize.spec.ts

```diff
@@ -359,6 +359,66 @@ describe('serializeNodeWithId', () => {
       )
     })
 
+    it('serializes <input type="radio"> elements checked state', () => {
+      const radio = document.createElement('input')
+      radio.type = 'radio'
+      expect(serializeNodeWithId(radio, DEFAULT_OPTIONS)! as ElementNode).toEqual(
+        jasmine.objectContaining({
+          attributes: {
+            type: 'radio',
+            value: 'on',
+            checked: false,
+          },
+        })
+      )
+
+      radio.checked = true
+
+      expect(serializeNodeWithId(radio, DEFAULT_OPTIONS)! as ElementNode).toEqual(
+        jasmine.objectContaining({
+          attributes: {
+            type: 'radio',
+            value: 'on',
+            checked: true,
+          },
+        })
+      )
+    })
+
+    it('serializes <input type="radio"> elements checked state with mask serialization', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**
* this test title is a bit misleading as it actually *does not* serialize the checked state.
* Maybe move it in the `describe('input privacy mode mask-user-input'` to colocate with similar tests